hr_employee_calendar_planning/i18n/es.po

```diff
@@ -6,15 +6,26 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 12.0\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2020-01-16 20:00+0000\n"
+"POT-Creation-Date: 2021-07-13 11:53+0000\n"
+"PO-Revision-Date: 2021-07-13 13:55+0200\n"
 "Last-Translator: Carles Antoli <carlesantoli@hotmail.com>\n"
 "Language-Team: none\n"
 "Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: \n"
+"Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Weblate 3.10\n"
+"X-Generator: Poedit 2.3\n"
+
+#. module: hr_employee_calendar_planning
+#: code:addons/hr_employee_calendar_planning/models/resource_calendar.py:0
+#, python-format
+msgid ""
+"%s is related to %s employee(s) in calendar planing, you should change it "
+"first."
+msgstr ""
+"%s está relacionado a %s empleado(s) en las horas laborales, deberías "
```

Comment:
This is not updated according latest code change, and use the formal verb: s/deberías/Debería